app/views/console/functions/function.phtml

```diff
@@ -316,6 +318,9 @@ $timeout = $this->getParam('timeout', 900);
                     <ul class="chart-notes margin-bottom-large">
                         <li class="red">Errors <span data-ls-bind="({{usage.failures.total}})"></span></li>
                     </ul>
+                    <?php else: ?>
+                        <p>Usage stats is disabled.</p>
+                    <?php endif; ?>
```

Comment:
Can you provide a screenshot of the dashboard? I'm not sure if we need to have a message on it or not. Let's discuss with the team once you can share the pictures.

Reply:
sorry, just seen the screenshots. Maybe you can remove the all <li> section, this will completely remove the `Monitors` tab.

Reply:
Ok

Reply:
have updated screenshot!